doc/ref_model/chapters/chapter03.md

```diff
@@ -543,33 +543,52 @@ SyncE architecture minimally requires replacement of the internal clock of the E
 ## 3.6 Storage
 The general function of storage subsystem is to provide the persistent data store required for the delivery of a network service. In the context of NFVi the storage sub-system needs to accommodate needs of: the tenanted VNF applications and the platform management.
 Storage is multi-faceted and so can be classified based on its: cost, performance (IOPS, throughput, latency), capacity and consumption model (platform native, network shared, object or archival) and the underlying implementation model (in chassis, software defined, appliance). A simplified view of this is provided in the following illustrative model:
-
+<p align="center"> <img src="../figures/rm-ch3.6-storage-model-02.png" alt="Storage Model - Cost vs Performance with Consumption Model" Title="Storage Model" width="45%"/></p>
+<p align ="center"><b>Figure 3-12:</b> Storage Model - Cost vs Performance with Consumption Model Overlay</p>
 
 Where:
-* Cost - is monetory value / unit of end user storage capacity
-* Performance - is IOPS / Latency / Throught Put as typically each of these increases with succesive genarations of storage
-* Storage Class - is how the storage is accessed and used, where: Platform Native = is managed by the hypervisor / platform (examples are a virtual disk volume from which a VNF  boots and can write back to, the storage interface that is exposed by the container runtime), this storage is typically not shared across running VNF / CNF instances; Shared Storage = is storage this accessed through a file systems interface (examples are network based storage such as CIFS or NFS) where the storage volumes can be accessed and shared by multiple VNF / CNF instances; Object Storage = is storage that is accessed via API interaces (the most common example being HTTP restful services API) whichs support get/put of structured objects; and Archival = is storage that is targetted with providing long term storage for purpose of disaster recovery, legal requirements or other historical recording where the storage mechanism my go through multiple stages before landing at rest.
-The storage model provides a relatively simple way for the storage consumer to specify / select their storage needs. This is shown in the follow table which highlights value of the storage classes and the epic use case for usage pattern.
+* (Comparitive) Cost - is monetory value / unit of end user storage capacity (see below for indicative comparative costs)
+* Performance - is defined by IOPS / Latency / Throught Put as typically each of these increases with succesive genarations of storage
+* Capacity - consumption needs are represented by width of the: Ultra High Performance, Enterprise Transactional, Value and Capacity storage options.
+* Storage Class - is how the storage is accessed and used, where:
```

Comment:
How about using "Storage Type" instead of "Storage Class"? The latter term is often used to differentiate (selectable) instances of the same storage type which exhibit different propoperties such as performance levels (e.g. gold, silver, bronze class).

Reply:
Noted and updated to "Storage Type", I will need to re-review to ensure all instance of "Storage Class" are updated correctly.